packages/realm-network-transport/tsconfig.json

```diff
@@ -3,19 +3,24 @@
 	"compilerOptions": {
 		"target": "es2018",
 		"forceConsistentCasingInFileNames": true,
+		"esModuleInterop": true,
+		"typeRoots": [
+			"./types",
+			"./node_modules/@types"
+		],
 		"types": [
 			"mocha",
 			"chai",
-			"express",
 			"node"
 		],
 		"lib": [
-			"ES2015",
-			"DOM"
+			"ES2015"
```

Comment:
Why not ES2018? Isn't that the default for the current target?

Also, we should leave trailing comments on lists (in files that support it like these) so that we don't get diffspam.

Reply:
It's just because we're not actively using features newer than ES2015 - if we did want to use those, we could simply bump this.